draft-ietf-privacypass-architecture.md

```diff
@@ -272,6 +272,9 @@ TokenChallenge --->|                   |
 Alternatively, when configured to do so, Clients may opportunistically present
 Token values to Origins without a corresponding TokenChallenge.
 
+The structure and semantics of the TokenChallenge and Token messages depend
+on the issuance protocol being used; see {{AUTHSCHEME}} for more information.
```

Comment:
Should we mention the token type here? Like:
```suggestion
on the issuance protocol and token type being used;
see {{AUTHSCHEME}} for more information.
```

Reply:
Good idea -- I made this change in both occurrences.